examples/cloud/tiledb_cloud_ml_model_array.ipynb

```diff
@@ -0,0 +1,228 @@
+    "our bucket on S3 (defined with AWS credentials in your TileDB-Cloud account) and register it on TileDB-Cloud."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Define array model uri.\n",
+    "uri = \"tiledb-sklearn-model\"\n",
+    "\n",
+    "print('Defining SklearnTileDB model...')\n",
+    "# In order to save our model on S3 and register it on TileDB-Cloud we have to pass our Namespace and TileDB Context.\n",
+    "tiledb_model = SklearnTileDB(uri=uri, namespace=TILEDB_USER_NAME, ctx=ctx)\n",
+    "\n",
+    "print(tiledb_model.uri)\n",
+    "\n",
+    "# We will need the uri that was created from our model class\n",
+    "# (and follows pattern tiledb://my_username/s3://my_bucket/my_array),\n",
+    "# in order to interact with our model on TileDB-Cloud.\n",
+    "tiledb_cloud_model_uri = tiledb_model.uri\n",
+    "\n",
+    "print('Saving model on TileDB-Cloud')\n",
+    "tiledb_model.save(\n",
+    "    model=clf, meta={\"Sparsity_with_L1_penalty\": sparsity, \"score\": score}\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "Finally, we can use TileDB-Cloud API as described in our [cloud documentation](https://docs.tiledb.com/cloud/), in order\n",
+    "to list our models, get information and deregister them."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# List all our models. Here, we filter with file_type = 'ml_model'. All machine learning model TileDB arrays are of type\n",
+    "# 'ml_model'\n",
+    "print(\n",
+    "tiledb.cloud.client.list_arrays(\n",
+    "    file_type=['ml_model'],\n",
+    "    namespace=TILEDB_USER_NAME))\n",
+    "\n",
+    "# Get model's info\n",
+    "print(tiledb.cloud.array.info(tiledb_cloud_model_uri))\n",
+    "\n",
+    "# Load our model for inference\n",
+    "loaded_tiledb_model = SklearnTileDB(uri=tiledb_cloud_model_uri, ctx=ctx).load()\n",
+    "\n",
+    "print(score == loaded_tiledb_model.score(X_test, y_test))\n",
+    "\n",
+    "# Deregister model\n",
+    "tiledb.cloud.deregister_array(tiledb_cloud_model_uri)"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.8"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 1
+}
```

Comment:
What is going on with the outputs?

Reply:
Removed because we don't have a demo account at the moment, and personal paths etc are shown when printing out array information etc.